tests/Unit/FormRequests/StoreUpdateRegistrationRequestTest.php

```diff
@@ -0,0 +1,209 @@
+                'passed' => false,
+                'data' => []
+            ],
+            'requestShouldFailWhenTooManyPriCarers' => [
+                'passed' => false,
+                'data' => [
+                    'pri_carer' => ['A String', 'B String'],
+                ]
+            ],
+            'requestShouldFailWhenNoPriCarers' => [
+                'passed' => false,
+                'data' => [
+                    'pri_carer' => [],
+                ]
+            ],
+            'requestShouldFailWhenNonStringPriCarers' => [
+                'passed' => false,
+                'data' => [
+                    'pri_carer' => [1],
+                ]
+            ],
+            'requestShouldSucceedWithSecondaryCarers' => [
+                'passed' => true,
+                'data' => [
+                    'pri_carer' => ['A String'],
+                    'sec_carers' => ['B String', 'C String'],
+                ]
+            ],
+            'requestShouldFailWithEmptySecondaryCarers' => [
+                'passed' => false,
+                'data' => [
+                    'pri_carer' => ['A String'],
+                    'sec_carers' => [],
+                ]
+            ],
+            'requestShouldFailWithNonStringSecondaryCarers' => [
+                'passed' => false,
+                'data' => [
+                    'pri_carer' => ['A String'],
+                    'sec_carers' => [1,2,3,4],
+                ]
+            ],
+            'requestShouldSucceedWithNewCarers' => [
+                'passed' => true,
+                'data' => [
+                    'pri_carer' => ['A String'],
+                    'new_carers' => ['B String', 'C String'],
+                ]
+            ],
+            'requestShouldFailWithEmptyNewCarers' => [
+                'passed' => false,
+                'data' => [
+                    'pri_carer' => ['A String'],
+                    'new_carers' => [],
+                ]
+            ],
+            'requestShouldFailWithNonStringNewCarers' => [
+                'passed' => false,
+                'data' => [
+                    'pri_carer' => ['A String'],
+                    'new_carers' => [1,2,3,4],
+                ]
+            ],
+            'requestShouldSucceedWithMinimalChildren' => [
+                'passed' => true,
+                'data' => [
+                    'pri_carer' => ['A String'],
+                    'children' => [
+                        0 => ['dob' => '2017-09']
+                    ],
+                ]
+            ],
+            'requestShouldFailWithChildInvalidDobFormat' => [
+                'passed' => false,
+                'data' => [
+                    'pri_carer' => ['A String'],
+                    'children' => [
+                        0 => ['dob' => '2017-09-01']
+                    ],
+                ]
+            ],
+            'requestShouldFailWithEmptyChildren' => [
```

Comment:
Maybe allow children to be deleted?

Reply:
As above, removing all children in an update does not submit the array.

Reply:
Thus, if the array is present, it MUST have one member